src/components/ClickableStyle/ClickableStyle.stories.tsx

```diff
@@ -209,12 +209,12 @@ IconClickableStyleIconOnlySmall.args = {
 };
 
 export const Link = Template.bind({});
-Link.args = { children: 'Clickable Style', variant: 'link' };
+Link.args = { children: 'Clickable Style', variant: 'linkDefault' };
```

Comment:
can totally table this discussion -- we got rid of the `ClickableStyle` stories in EDS `main`, do we want them back now?